pandas/sparse/frame.py

```diff
@@ -302,7 +302,7 @@ def fillna(self, value=None, method=None, axis=0, inplace=False,
     # ----------------------------------------------------------------------
     # Support different internal representation of SparseDataFrame
 
-    def _sanitize_column(self, key, value):
+    def _sanitize_column(self, key, value, broadcast=False):
```

Comment:
@jorisvandenbossche sure. I think fixing the behavior of the sparse data frame here is out of scope for this PR.